### PR TITLE
chore(manifest): parse strings using shell-style rules

### PR DESCRIPTION
**Pull request by Lou1415926**
In #1998, the way that the strings are split into string slice may cause bugs when passing the values to cfn template. This PR implements a similar fix as #1467.

Fixes #1998 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: internal/pkg/manifest/workload.go [hunk 1]
```diff
@@ -7,9 +7,10 @@ package manifest
 import (
 	"errors"
 	"fmt"
+	"github.com/google/shlex"
 	"path/filepath"
 	"strconv"
-	"strings"
+	//"strings"
```
Comment (iamhopaul123) [inline internal/pkg/manifest/workload.go]: nit: can we remove this?

**Review by iamhopaul123 — APPROVED** [review]
LGTM :shipit:

File: internal/pkg/manifest/workload.go (hunk 1 above)
Comment (Lou1415926) [reply internal/pkg/manifest/workload.go]: lol i'm blind

File: internal/pkg/deploy/cloudformation/stack/backend_svc.go [hunk 2]
```diff
@@ -87,6 +87,14 @@ func (s *BackendService) Template() (string, error) {
 	if err != nil {
 		return "", fmt.Errorf("convert storage options for service %s: %w", s.name, err)
 	}
+	entrypoint, err := s.manifest.EntryPoint.ToStringSlice()
+	if err != nil {
+		return "", fmt.Errorf("convert entrypoint to string slice: %w", err)
```
Comment (efekarakus) [inline internal/pkg/deploy/cloudformation/stack/backend_svc.go]: nit (same for the other errors):
```suggestion
		return "", fmt.Errorf(`convert 'entrypoint' to string slice: %w`, err)
```

**Review by efekarakus — APPROVED** [review]
Looks good to me! just a small error message improvement. Feel free to remove the "do-not-merge" label after addressing it. Mergify will then automatically merge the PR :)

**Pull request merged**